File: Notification-Service/src/test/java/byrd/riley/notificationservice/test/model/QueueSender.java
package byrd.riley.notificationservice.test.model;

import org.springframework.amqp.rabbit.test.TestRabbitTemplate;

public class QueueSender {

	private TestRabbitTemplate testRabbitTemplate;
	
	public QueueSender(TestRabbitTemplate testRabbitTemplate) {
		this.testRabbitTemplate = testRabbitTemplate;
	}

    public void send(String receiverPhone, String messageBody) {
        String message =
    		"text\r\n" +
    		receiverPhone + "\r\n" +
    		messageBody;
        testRabbitTemplate.convertAndSend(TestNotificationConfiguration.QUEUE_NAME, message);
        System.out.println(" [x] Sent '" + message + "'");
    }
}

File: Notification-Service-Kotlin/src/test/kotlin/byrd/riley/notificationservicekotlin/test/model/QueueSender.kt
package byrd.riley.notificationservicekotlin.test.model

import org.springframework.amqp.rabbit.test.TestRabbitTemplate

class QueueSender(private val testRabbitTemplate: TestRabbitTemplate) {

    fun send(receiverPhone: String, messageBody: String) {
        val message =
            "text\r\n" +
            "${receiverPhone}\r\n" +
            messageBody

        testRabbitTemplate.convertAndSend(TestSMSConfiguration.QUEUE_NAME, message)
        print(" [x] Sent '${message}'")
    }
}